tls/s2n_connection.c

```diff
@@ -563,6 +557,9 @@ int s2n_connection_wipe(struct s2n_connection *conn)
     POSIX_GUARD(s2n_stuffer_wipe(&conn->header_in));
     POSIX_GUARD(s2n_stuffer_wipe(&conn->in));
     POSIX_GUARD(s2n_stuffer_wipe(&conn->out));
+    /* free and wipe the cookie stuffer since it is reallocate below */
+    POSIX_GUARD(s2n_stuffer_free(&conn->cookie_stuffer));
+    POSIX_GUARD(s2n_stuffer_wipe(&conn->cookie_stuffer));
```

Comment:
is the s2n_stuffer_wipe call here necessary? You've already freed, and the memory will get zeroed later in the method.

It also looks like we usually just use [s2n_stuffer_resize](https://github.com/aws/s2n-tls/blob/main/stuffer/s2n_stuffer.c#L126-L131) for growable stuffers (see later in this method), so you could use that for consistency.